src/mca/ploc/hwloc/ploc_hwloc.c

```diff
@@ -683,22 +689,8 @@ pmix_status_t setup_topology(pmix_info_t *info, size_t ninfo)
                                         &shmemaddr, shmemsize)) {
         /* we couldn't find a hole, so don't use the shmem support */
         if (4 < pmix_output_get_verbosity(pmix_ploc_base_framework.framework_output)) {
-            FILE *file = fopen("/proc/self/maps", "r");
-            if (file) {
-                char line[256];
-                pmix_output(0, "%s Dumping /proc/self/maps",
-                            PMIX_NAME_PRINT(&pmix_globals.myid));
-                while (fgets(line, sizeof(line), file) != NULL) {
-                    char *end = strchr(line, '\n');
-                    if (end) {
-                       *end = '\0';
-                    }
-                    pmix_output(0, "%s", line);
-                }
-                fclose(file);
-            }
+            return ploc_hwloc_print_maps();
         }
-        return PMIX_SUCCESS;
```

Comment:
Don't we still want to return `PMIX_SUCCESS` here in the case where verbosity is not set?

Reply:
Yes, this is right. Thanks!

Fixed it.